index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Web-site</title>

```

Comment:
Please use a valid web-site title according to the name of the project. This title is confusing, not clear and it doesn't say to a user what is the name of the web-site.